### PR TITLE
Pill filter list

### PR DESCRIPTION
**Pull request by mjoyce91**
This PR adds container to Results page with list of pills representing the currently selected filters. When clicked on, the pill and related filter are removed and the page refreshes.

Relies on #404, #439, #445, #449.

### CODE REVIEW

File: src/Components/PillList/PillList.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import PropTypes from 'prop-types';
+import { PILL_ITEM_ARRAY } from '../../Constants/PropTypes';
+import Pill from '../Pill/Pill';
+
+const shortid = require('shortid');
+
+const PillList = ({ items, onPillClick }) => (
+  <div>
+    {
+      (items.sort((a, b) => { // sort our pills by description
```
Comment (burgwyn) [inline src/Components/PillList/PillList.jsx]: Can we pull the sort logic out of `render`?  Maybe push it further up the tree?

Comment (jseppi) [reply src/Components/PillList/PillList.jsx]: Might also want to add a `.slice()` before the `.sort(...)` so that the original array isn't mutated.

Comment (mjoyce91) [reply src/Components/PillList/PillList.jsx]: Thanks! I resolve this in #456

**Comment by mjoyce91** [comment]
Resolved by #456

**Pull request closed without merging**